backend/internal/test/integration/helper/envoytest/envoy.go

```diff
@@ -69,7 +69,7 @@ type EnvoyHandle struct{}
 
 // NewEnvoyHandle creates a new handle for the Envoy under test after waiting for it to initialize.
 func NewEnvoyHandle() (*EnvoyHandle, error) {
-	timeout := time.NewTimer(5 * time.Second)
+	timeout := time.NewTimer(40 * time.Second)
```

Comment:
I don't think we need this long of a timeout?

Reply:
In fact, I had to increase the timeout to 60 seconds. Not sure where the time is used